src/ChangeStream.php

```diff
@@ -262,7 +262,7 @@ private function onIteration(bool $incrementKey): void
      */
     private function resume(): void
     {
-        if (! $this->resumeCallable) {
+        if (null === $this->resumeCallable) {
```

Comment:
Not sure this is strictly necessary, but if we're going to not use the shorthand then let's not use Yoda style ;)

```suggestion
        if ($this->resumeCallable === null) {
```